utils/bump_version.py

```diff
@@ -24,7 +24,8 @@
 
 
 def stringify_version(
-    version_info: VersionInfo, in_develop: bool = True,
+    version_info: VersionInfo,
```

Comment:
Just wondering, but did it put this on one two lines because we disabled the trailing comma?

Reply:
Ha this is a little ideocyncracy, that black/ruff always "respects" existing trailing-commas, explained here: https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#the-magic-trailing-comma

I've just made a commit, removing the trailing commas, and you can see the signatures are now on one line

You can remove this behaviour with https://docs.astral.sh/ruff/settings/#format_skip-magic-trailing-comma

Reply:
So, to respect existing trailing commas or not?

Reply:
So... should we change the behaviour? As I said before, it's a personal taste but if we are running ruff format then I won't complain to whatever it decides to do. However, if it's possible to avoid adding lines to files that are sometimes already large, I'd be happy (and stay consistent with the rules we have).

Reply:
(Actually, I wasn't expecting you to change the lines)

Reply:
> So... should we change the behaviour?

well I guess what kinda makes sense is that, the first time you run the formatting on an un-formatted file, you use `skip-magic-trailing-comma = false`, but then in general you just run with the default `skip-magic-trailing-comma = true`

anyway, with the changes I just made it now stays the same whether you use `true` or `false`

Reply:
I agree with that. Maybe you can open an issue for ruff formatting the code base so that contributors know about it.